public/ThumbsRating.js

```diff
@@ -0,0 +1,37 @@
+// Allows user to select a thumbs up or thumbs down rating to be stored with their entry.
+import html from './html.js';
+
```

Comment:
Also, should this file be in the `/packages/frontend/public` folder instead of the `/public` folder?